hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -147,11 +147,13 @@ vcgencmd otp_dump | grep 28: | sed s/.*://g
 ### Installation - firmware update
 Network boot functionality is included in the 2020-02-13 Raspbian Buster release. However, for advanced boot modes it's normally best to use the latest stable software.
 ```
-# Install the rpi-eeprom update package                                                                                                 
+# Update the rpi-eeprom update package                                                                                                 
```

Comment:
"rpi-eeprom update" -> "rpi-eeprom-update" ?

Reply:
The package name is actually rpi-eeprom

Reply:
Ahhh. Then maybe this should just say "Update the rpi-eeprom package" ?